### ISSUE TITLE
Do you consider a new release

### ISSUE THREAD

**Issue by diki** [Hacker0x01/react-datepicker#98]
Thank you for this, it is the best datepicker so far i found.

I installed v0.6.0 through npm and popover was not working fine. Problem was calendar popped on top of inputs if it is 6 line long. I realize problem is about tether options on popover.jsx, while i was trying to fix found out that you also changed tether options but did not release. So i forked and used own release it works much better now.

TL;DR: popover position problem is fixed in source but not is latest release.

**Comment by RSO** [comment]
Ah, sorry, must have forgotten to do one! I'll get on it right now!

**Comment by RSO** [comment]
Should be fixed! Thanks for the compliment!
